R/getDegreeDays.R

```diff
@@ -210,10 +235,35 @@ getDegreeDays <- function(mappingFile = NULL,
   # extract all job IDs
   jobIds <- lapply(allJobs, function(x) x$jobId)
 
-  # wait for our specific jobs to complete (max. 12hrs)
-  waitForSlurm(jobIds, maxWaitTime = 12 * 60 * 60)
+  # wait for our specific jobs to complete (max. 6hrs)
+  waitForSlurm(jobIds, maxWaitTime = 6 * 60 * 60)
+
+
+
+  # --- GATHER AND SMOOTH DEGREE DAYS
+
+  # gather outputs from slurm jobs
+  data <- gatherData(fileMapping = fileMapping, outDir = outDir)
+
+  # smooth degree days
+  dataSmooth <- smoothDegreeDays(data, nSmoothIter = 50, transitionYears = 5)
+
+
+
+
+  # OUTPUT ---------------------------------------------------------------------
+
+  fileName <- "hddcdd"
+
+  if (is.character(fileRev)) {
+    fileName <- paste0(fileName, "_", fileRev)
+  } else if (!is.null(fileRev)) {
+    warning("fileRev must be character")
+  }
+
+  outPath <- file.path(outDir, paste0(fileName, ".csv"))
 
+  write.csv(dataSmooth, outPath, row.names = FALSE)
 
 
```

Comment:
How about a silent return? This can be helpful and shouldn't hurt if you don't use it.
```suggestion
return(invisible(outPath))
```